Source/WebInspectorUI/UserInterface/Controllers/NetworkManager.js

```diff
@@ -1607,6 +1703,16 @@ WI.NetworkManager = class NetworkManager extends WI.Object
     }
 };
 
+WI.NetworkManager.EmulatedCondition = {
```

Comment:
I dont believe we've ever done a `WI.UIString` on load.  It should theoretically be possible, but I'll have to check and make sure.

Most of this was just to avoid having to do all that work on load, but it's probably a drop in the bucket.  I'll rework.